materials/intro/async/static/actor.py

```diff
@@ -0,0 +1,150 @@
+"""This file has been placed in the public domain"""
+import asyncio
+import contextlib
+import time
+
```

Comment:
`intro/async/static/actor.py` was removed since it's no longer used. Please don't re-add it.